perf/jmh/run_jmh_jdk_micros.sh

```diff
@@ -0,0 +1,249 @@
+#!/bin/bash
+
+# This script is based on one of Monica Beckwith's scripts.
+function help
+{
+    echo -e "\nUsage: $0 [GC HeapSizeGB WorkerThreads WarmupTimeSeconds WarmupIterations MeasurementIterations [JvmArgs]]\n"
+    echo "Example: $0 Parallel 80 1 10 5 5"
+    echo "Example: $0 g1       80 1 10 5 5 -XX:MaxGCPauseMillis=30 -XX:MaxNewSize=30g"
+}
+source run/testenv.mk
```

Comment:
Perhaps move this "include" to top of the file.

Reply:
I think that's redundant. I will get rid of it.